rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -238,13 +234,8 @@ public Response startLRA(
                     return Response.status(response.getStatus()).build();
                 }
             } else {
-                ResponseHolder resp = new RequestBuilder(parentLRAUrl)
```

Comment:
You can't just ignore code.
Are there any other places in this PR where you have removed stuff that's required?

Reply:
This was something that I actually thought was an oversight as this code calls the same method os the other if branch ([L235](https://github.com/jbosstm/narayana/pull/1677/files#diff-2c664918d9eca78964a0ea919711c8aaL235)) just through JAX-RS invocation which doesn't make sense if the parent LRA id is not found. Would you prefer I take this in a new JBTM?

Reply:
The if part of the statement is a local invocation.
The else part of the statement is a remote invocation.

ie they execute different behaviours so you can't just remove the else part (we need to interact with remote coordinators since this is not a centralised service - we're operating in distributed microservices based environment. 

I am not aware of any bug in the original code so there's no need for a JBTM, unless you have a test case that shows there is a problem with it.

Reply:
I vaguely recall either a demo or a quickstart that relied on multiple coordinators. Check the quickstarts (in fact I think you or someone else needs to verify that the quickstarts still run with these updates before we attempt merging them).

Reply:
In particular the quickstarts rely on thorntail. As we discussed earlier replacing ApacheHttp client calls with JAX-RS thorntail does not work without your ForkJoinClassLoading fix. But running the quickstarts would validate that fix.